furms-model-api/src/main/java/io/imunity/furms/api/validation/exceptions/ProjectHasMoreThenOneResourceTypeAllocationInGivenTimeException.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2021 Bixbit s.c. All rights reserved.
+ *  See LICENSE file for licensing information.
+ */
+
+package io.imunity.furms.api.validation.exceptions;
+
+public class ProjectHasMoreThenOneResourceTypeAllocationInGivenTimeException extends IllegalArgumentException {
+    public final String projectId;
+    public final String resourceTypeId;
+
+    public ProjectHasMoreThenOneResourceTypeAllocationInGivenTimeException(String projectId, String resourceTypeId) {
+        this.projectId = projectId;
```

Comment:
call super with "projectId: " + projectId + " resourceTypeId: " + resourceTypeId
This will give meaningful information in the logs when this happens.
Do a test: log this exception with and without calling super